src/python/py_whisper_pipeline.cpp

```diff
@@ -11,7 +11,7 @@
 #include "openvino/genai/whisper_generation_config.hpp"
 #include "openvino/genai/whisper_pipeline.hpp"
 #include "py_utils.hpp"
-#include "tokenizers_path.hpp"
+#include "tokenizer/tokenizers_path.hpp"
```

Comment:
How about removing duplicate words from paths? tokenizer/tokenizers_path.hpp -> tokenizer/path.hpp?
It also seems that continuous_batching_adapter.hpp shouldn't be part of continuous_batching because continuous_batching itself is fine without the adapter.

I'm fine submitting a PR to you if we agree on that

Reply:
done. 

Please, check and provide suggestion about file naming / renaming if any